atomate/vasp/fireworks/core.py

```diff
@@ -831,3 +833,176 @@ def __init__(self, spec, neb_label, from_images=True,
+                 shift=None, ssize=None, reconstruction=None,  **kwargs):
+        """
+        Initializes the Firework.
+
+        Args:
+            structure (Structure): Can be a conventional unit cell, oriented
+                unit cell or slab cell.
+            scratch_dir (str): - if specified, uses this directory as the root
+                scratch dir. Supports env_chk.
+            k_product (int): Default to 50, kpoint number * length for a & b
+                directions, also for c direction in bulk calculations.
+            db_file (str): FULL path to file containing the database credentials.
+                Supports env_chk.
+            vasp_cmd (str): Command to run vasp.
+            structure_type (str): Type of structure. Options are:
+                conventional_unit_cell, oriented_unit_cell and slab_cell
+            miller_index ([h, k, l]): Miller index of plane parallel to
+                surface (and oriented unit cell).
+            ouc (Structure): The oriented_unit_cell from which
+                this Slab is created (by scaling in the c-direction).
+            shift (float): The shift in the c-direction applied to get the
+                termination.
+            ssize (float): Minimum slab size in Angstroms or number of hkl planes
+            vsize (float): Minimum vacuum size in Angstroms or number of hkl planes
+            mmi (int): Max Miller index.
+            scale_factor (array): Final computed scale factor that brings
+                the parent cell to the surface cell.
+            mpid (str): Materials Project ID of the conventional unit cell.
+            reconstruction (str): The name of the reconstruction
+                (if it is a reconstructed slab).
+            cwd (str): Location of directory to operate the
+                workflow and store the final outputs
+            \*\*kwargs: Other kwargs that are passed to Firework.__init__.
+        """
+
+        self.structure = structure.copy()
+        self.scratch_dir = scratch_dir
+        self.el = self.structure[0].species_string
+        self.structure_type = structure_type
+        self.k_product = k_product
+        self.mpid = mpid
+        self.hkl = miller_index
+        self.reconstruction = reconstruction
+        self.ssize = ssize
+        self.vsize = vsize
+        self.shift = shift
+        self.sg = SpacegroupAnalyzer(ouc if structure_type ==
+                                            "slab_cell" else structure)
+        self.scale_factor = scale_factor
+        self.ouc = ouc
+        self.cwd = cwd
+        self.vasp_cmd = vasp_cmd
+        self.db_file = db_file
+        self.mmi = mmi
+
+        super(SurfCalcOptimizer, self).__init__(self.get_tasks,
+                                                name=self.get_name, **kwargs)
+
+    @property
+    def get_input_set(self):
+
+        # Get the MVLSlabSet based on structure type
+        # Returns input_set
+
+        if self.structure_type != "slab_cell":
```

Comment:
This is a pretty minor point, but all else being equal, I usually find it more readable to have "if" statements be positive rather than negative.

i.e.,

```
if self.structure_type == "slab_cell":
    do something
else:
   do something else
```